Orange/widgets/tests/test_owselectcolumns.py

```diff
@@ -105,3 +107,33 @@ def retrieveSpecificSettings(self):
 
     def storeSpecificSettings(self):
         pass
+
+class TestOWSelectAttributes(WidgetTest):
+    def setUp(self):
+        self.widget = self.create_widget(OWSelectAttributes)
+
+    def testall(self, location):
+        self.assertEqual(len(self.widget.available_attrs), location[0])
+        self.assertEqual(len(self.widget.used_attrs), location[1])
+        self.assertEqual(len(self.widget.class_attrs), location[2])
+
+    def test_multiple_target_variable(self):
+        '''
+        A box for Target variables is too small.
+        When one target variable is moved somewhere else, it cannot be moved back
+        if there is already one target variable.
+        GH-2100
+        GH-2086
+        '''
+        iris = Table("iris")
+        self.send_signal("Data", iris)
+        self.widget.move_class_button.click()
```

Comment:
It might be a good idea to add an assert before anything is clicked. The main point would be to show what the variable counts are when the widget is opened.